tests/installation/installation_overview.pm

```diff
@@ -42,6 +42,9 @@ sub ensure_ssh_unblocked {
                 send_key_until_needlematch 'ssh-blocked-selected', 'tab', 25;
                 send_key 'ret';
                 send_key_until_needlematch 'ssh-open', 'tab';
+                send_key_until_needlematch 'ssh-service-disabled', 'tab', 25;
+                send_key 'ret';
+		send_key_until_needlematch 'ssh-service-enabled', 'tab';
```

Comment:
ah, sure, I missed a parenthesis :facepalm: 

```suggestion
                if (check_var('ENABLE_SSHD_BEFORE_INSTALL', 1)) {
                  send_key_until_needlematch 'ssh-service-disabled', 'tab', 25;
                  send_key 'ret';
                  send_key_until_needlematch 'ssh-service-enabled', 'tab';
                }
```

Reply:
So I have to add ENABLE_SSHD_BEFORE_INSTALL=1 into the openQA configuration of gpt?

Reply:
Then the ssh-service-disabled can be skipped...

Reply:
> So I have to add ENABLE_SSHD_BEFORE_INSTALL=1 into the openQA configuration of gpt?

no, that would put that setting for any arch and medium that execute gpt test suite. Let me add it in YAML job group. We can remove it later if it was a bad idea.

Reply:
added: https://openqa.opensuse.org/admin/job_templates/38

```
  s390x:
    opensuse-Tumbleweed-DVD-s390x:
    - ...
    - gpt:
        settings:
          ENABLE_SSH_BEFORE_INSTALL: '1'
```

Reply:
> Then the ssh-service-disabled can be skipped...

I think it is good to have it. This will help for future reviewers to understand the intention and what's happening.